src/cli-command.ts

```diff
@@ -408,6 +416,42 @@ export class Command {
     }
   }
 
+  static loginToContainer(container: string, podName: string, namespace: string): CliCommand {
+    if (ocFallBack.get('ocFallBack')) {
+      return newOcCommand('exec', '-it', '-n', namespace, '-c', container, podName, 'bash');
```

Comment:
I receive message
`kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead`.
Could you verify that, and use new type of the command

Reply:
okay I will use ```kubectl exec mypod -i -t```

Reply:
@evidolob done